transaction/tests/LockRequest_unittest.cpp

```diff
@@ -41,7 +41,7 @@ class LockRequestTest : public ::testing::Test {
 TEST_F(LockRequestTest, CheckGetters) {
   EXPECT_EQ(transaction_id(3), lock_request_.getTransactionId());
   EXPECT_EQ(ResourceId(5), lock_request_.getResourceId());
-  EXPECT_EQ(AccessMode(AccessModeType::kSLock), lock_request_.getAccessMode());
+  EXPECT_EQ(AccessMode(AccessMode::SLockMode()), lock_request_.getAccessMode());
```

Comment:
Refactor to `EXPECT_EQ(AccessMode::SLockMode(), lock_request_.getAccessMode());`